src/core/inbox.rs

```diff
@@ -93,13 +95,9 @@ mod axum_imp {
             + From<Error>
             + From<<ActorT as ApubObject>::Error>
             + From<serde_json::Error>,
-        // FIXME
         // + From<http_signature_normalization_actix::digest::middleware::VerifyError>,
         <ActorT as ApubObject>::Error: From<Error> + From<anyhow::Error>,
     {
-        // ensure that payload hash was checked against digest header by middleware
-        /// FIXME
-        // DigestVerified::from_request(&request, &mut Payload::None).await?;
```

Comment:
This is only necessary in the actix code to avoid security problems in case a library user forgets to use the actix middleware for signature verification. It would be better to make the signature check directly in the library, and then remove the actix VerifyDigest middleware as well as DigestVerified